docs/user-defined-type-functions.md

```diff
@@ -110,13 +110,13 @@ This section details the initial programming interface we propose for `type`s wh
 
 | Static Methods | Return Type | Description |
 | ------------- | ------------- | ------------- |
-| `get(arg: string \| boolean \| nil)` | `type` | returns an immutable runtime representation of the argument as a singleton type |
-| `negationof(arg: type)` | `type` | returns an immutable runtime representation of the argument as a negated type; the argument cannot be an instance of a table or a function |
-| `unionof(...: type)` | `type` | returns an immutable runtime representation of an union type of the arguments; requires the arguments size to be at least 2 |
-| `intersectionof(...: type)` | `type` | returns an immutable runtime representation of an intersection type of the arguments |
-| `newtable(props: {[type]: type}?, indexer: {key: type, value: type}?, metatable: type?)` | `type` | returns a mutable runtime representation of a table type; the keys of the table's property must be a runtime representation of the singleton types; if provided the metatable parameter, this table becomes a metatable |
-| `newfunction(parameters: {pack: {type}?, variadic: type?}, returns: {pack: {type}?, variadic: type?})` | `type` | returns a mutable runtime representation of a `function` type |
-| `copy(arg: type)` | `type` | returns a deep copy of the argument |
+| `singleton(arg: string \| boolean \| nil)` | `type` | returns an immutable instance of the argument as a singleton type |
```

Comment:
I like `scalar` the best, but fine with this name if we want to keep it.

Reply:
`singleton` is what `is` checks against as a string.